src/components/Divider/divider.module.css

```diff
@@ -0,0 +1,20 @@
+:root {
+  --denhaag-divider-border-color: var(--denhaag-grey-2);
+  --denhaag-divider-border-width: 1px;
+  --denhaag-divider-margin: 16px 16px;
+  --denhaag-divider-vertical-margin-inline: 16px 16px;
```

Comment:
The tokens should only be defined in the `styles/components/src/tokens-divider.css` file (with a few exceptions). Some are double now, exisiting both in the component tokens file and in the component css here.